oak_containers_orchestrator/src/container_runtime.rs

```diff
@@ -131,16 +132,23 @@ pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
     };
     rmount_dir(std::path::PathBuf::from("/proc"), container_proc_path).await?;
 
-    // Start the trusted application in a chroot jail of the container.
+    // Change the orchestrator's root to the newly created container rootfs.
+    // This will ensure that the newly launched process will also exist in
+    // this chroot jail.
+    // Note though that following this command, the orchestrator will no longer
+    // be able to access files outside of the chroot jail. This is not a problem
+    // given the orchestrators current feature set, but may become one later.
+    std::os::unix::fs::chroot(container_rootfs_path)?;
```

Comment:
If we want to continue running the orchestrator outside of the chroot jail, we can build a second binary that's solely responsible for setting the chroot & working dir before switching to the trusted app.